cli/dstack/utils/jinjamock.py

```diff
@@ -0,0 +1,27 @@
+class JinjaMockValue:
+    def __init__(self, s: str):
+        self.__s__ = s
+
+    def __repr__(self):
+        return self.__s__
+
+    def __getattr__(self, item):
+        raise ValueError()
+
+    def __getitem__(self, item):
+        raise ValueError()
+
+
+class JinjaMock:
+    """
+    Example:
+        value = "${{ secrets.any_key }}"
+        assert value == Template(value, variable_start_string="${{").render(secrets=JinjaMock("secrets"))
+    """
+
+    def __init__(self, namespace: str):
+        self.__n__ = namespace
+
+    def __getattr__(self, item: str) -> JinjaMockValue:
+        namespace = self.__dict__["__n__"]
```

Comment:
Seem like some weird magic. Why can't we just store `self.namespace = namespace`? Why access via self.__dict__? Why return `JinjaMockValue`? Can't we just return a string? Why name variables like __s__ – i.e. double underscores on both sides usually means it has special meaning to the interpreter.

Not critical, just reads weird.

Reply:
I tried to invent something unlikely to be used as a secret name

`${{ secrets.namespace }}` is more likely than `${{ secrets.__n__ }}` in my opinion, but brokes `JinjaMock`

`JinjaMockValue` is designed to raise errors in indexing and chaining. `str` would allow indexing and break `JinjaMock`: `${{ secrets.password[1] }}` and `${{ secrets.password.hash }}`

Probably I could avoid using `__dict__` there

Reply:
`JinjaMock` is removed